getnotes/getnotes.py

```diff
@@ -414,8 +422,65 @@ async def findplayer(self, ctx, *,player: Union[ipaddress.IPv4Address, int, str]
         
         except ModuleNotFoundError:
             return await message.edit(content="`mysql-connector` requirement not found! Please install this requirement using `pip install mysql-connector`.")
-                      
+
+
+    async def dph(self, ctx) -> dict: # Deaths per hour
+        """
+        Calculate a player's deaths per hour
+        """
+        player = key_to_ckey(player)
+
+        try:
+            message = await ctx.send("Calculating DPH....")
+            async with ctx.typing():
+                player = await self.player_search(ctx, ckey=player)
+            
+            if player is None:
+                raise ValueError
+
+            dph = round(player['num_deaths'] / (player['living_time'] / 60), 2)
```

Comment:
should be encased with a try/except to capture ZeroDivisionError exceptions.
```suggestion
            try:
                dph = round(player['num_deaths'] / (player['living_time'] / 60), 2)
            except ZeroDivisionError:
                dph = 0
```